lib/src/Uncertainty/Distribution/KernelSmoothing.cxx

```diff
@@ -256,8 +258,41 @@ Distribution KernelSmoothing::build(const Sample & sample) const
 {
   // For 1D sample, use the rule that give the best tradeoff between speed and precision
   if (sample.getDimension() == 1)
-    return build(sample, computeMixedBandwidth(sample));
-
+  {
+    if (useLogTransform_)
+    {
+      const Scalar skewness = sample.computeSkewness()[0];
+      const Scalar xMin = sample.getMin()[0];
+      const Scalar xMax = sample.getMax()[0];
+      const Scalar delta = (xMax - xMin) * std::max(SpecFunc::Precision, ResourceMap::GetAsScalar("KernelSmoothing-DefaultShiftScale"));
+      SymbolicFunction transform;
+      SymbolicFunction inverseTransform;
+      OSS oss(true);
+      // To get the full double precision
+      oss.setPrecision(17);
+      if (skewness >= 0.0)
+      {
+        const Scalar shift = delta - xMin;
+        transform = SymbolicFunction("x", String(oss << "log(x+(" << shift << "))"));
```

Comment:
The code is ok, but it seems to me that a SymbolicFunction taking "x" and "shift" as inputs would be clearer. Then a `ParametricFunction` setting the "shift" parameter to the value computed earlier would be clearer.

Reply:
It has been fixed, using a ParametricFunction instead of an inlined value for shift